src/Symfony/Component/Ldap/Tests/LdapClientTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Tests\Authentication\Provider;
+
+class LdapClientTest extends \PHPUnit_Framework_TestCase
+{
+}
```

Comment:
Will be implemented as we need to test the output of the shim for ldap_escape, as mentioned to @lyrixx yesterday.